integration-tests/tests/common/mod.rs

```diff
@@ -723,6 +723,11 @@ impl<'a> TestServerConfigurator<'a> {
                     "bind",
                     &format!("127.0.0.1:{}", self.server_number.server_port().unwrap()),
                 );
+                cfg.insert("owner_port", &self.server_number.server_port().unwrap());
```

Comment:
Maybe call this server_port to be consistent?

Reply:
uhm, not sure I understand :thinking:

Reply:
Basically, this is inserted always, also with rendezvous server.
Maybe just make this call `cfg.insert("server_port", &self.server_number.server_port().unwrap());`?

Reply:
> Basically, this is inserted always, also with rendezvous server.

is it? I see that only in rendezvous-info.yml in the tests :thinking:

Reply:
I think it's inserted with specific ports when calling generate_config_file so not within a server creation where it can inject its own port